src/systemctl/systemctl.c

```diff
@@ -4594,7 +4594,7 @@ static int show_one(
         if (r < 0)
                 return log_error_errno(r, "Failed to map properties: %s", bus_error_message(&error, r));
 
-        if (streq_ptr(property_info.load_state, "not-found") && streq_ptr(property_info.active_state, "inactive"))
+        if (show_properties && streq_ptr(property_info.load_state, "not-found") && streq_ptr(property_info.active_state, "inactive"))
```

Comment:
Why the check for `show_properties` here? Why should the "systemctl status" and "systemctl show" case be different here?